security/cmd/node_agent_k8s/nodeagent.yaml

```diff
@@ -40,4 +40,4 @@ spec:
         - name: flexvol-driver-host
           hostPath:
             type: DirectoryOrCreate
-            path: /home/kubernetes/flexvolume
+            path: /home/kubernetes/flexvolume/nodeagent~uds
```

Comment:
Is `~` valid in the FS path?

Reply:
yes, it's actually purposed designed by flexvolume: https://screenshot.googleplex.com/xicQCYS7N3a